test/pad_table.md

```diff
@@ -0,0 +1,26 @@
+# This is a test document
```

Comment:
I agree that looking for '|' is not the best idea I have had. The tags did not strike me. I'll do that in a while since I'm travelling right now.